data-exporter/README.md

```diff
@@ -0,0 +1,61 @@
+# Data Exporter (post-release)
+
+This program will produce data exports for submission to NCBI, UCSC, and Europe PMC.  Files provided to these resources are:
+* **NCBI:** Gene (NCBI gene identifier to Reactome top level pathways) and Protein (all UniProt entries in Reactome associated with any NCBI Gene identifier).
+* **UCSC:** Entity (UniProt entires in Reactome) and Events (UniProt entries in Reactome in relation to Reactome pathways and reactions).
```

Comment:
*entries

Reply:
I keep doing that :)!  I'll correct the spelling.